api/workAllocation/caseWorkerService.ts

```diff
@@ -6,7 +6,14 @@ import { setHeaders } from '../lib/proxy';
 import { CaseworkerPayload, ServiceCaseworkerData } from './interfaces/caseworkerPayload';
 
 const logger: JUILogger = log4jui.getLogger('caseworker-service');
-const MAX_RECORDS: number = 10000;
+const MAX_RECORDS: number = 100000;
```

Comment:
Is there evidence that you need to go to 100K? Can we go to 20K or 30K initially. Another option is to make multiple calls for 10K - API supports pagination.

Reply:
As discussed on standup, the current number is suitable for the moment but may be looked at in future